src/CONST.js

```diff
@@ -362,7 +362,7 @@ const CONST = {
             PAYPAL_ME: 'PayPal.me',
             VENMO: 'Venmo',
         },
-        AMOUNT_MAX_LENGTH: 14,
+        AMOUNT_MAX_LENGTH: 10,
```

Comment:
So as I explained [here](https://github.com/Expensify/App/pull/5243#discussion_r708523399), we always use amounts in cents in the Expensify backend, so if for example you type `$1.23`, then we use the amount `123`. If you type `$20`, then we send `2000`.
The amount in cents is what needs to be 10 digits max, so we need to multiply the amount to check in `validateAmount()` by 100 first. 

Does that make sense?

Reply:
Okay, I somehow misunderstood. I've updated the PR with the new cent logic.

Reply:
Limits are 8 digits without decimal and 10 digit with decimal.

<img width="331" alt="Screenshot 2021-09-15 at 11 29 47 PM" src="https://user-images.githubusercontent.com/57435789/133485464-95c940a0-4fa0-4fe8-9d43-180c71c5d980.png">

<img width="338" alt="Screenshot 2021-09-15 at 11 29 39 PM" src="https://user-images.githubusercontent.com/57435789/133485450-6103ef98-67c0-4a0f-a643-f1faa73dc267.png">